web-components/src/components/inputs/FieldFormControl.tsx

```diff
@@ -10,7 +11,15 @@ interface RenderProps {
   handleBlur: (value: any) => void;
 }
 
-interface Props extends FieldProps {
+export interface DefaultStyleProps {
+  defaultStyle?: boolean;
```

Comment:
It feels a bit odd to have a `default` prop, but I trust that you came to this solution for a good reason.

Reply:
this `defaultStyle` is just there to be able to set it to false in case you don't want the default styles to be applied